content/docs/04-clusters/01-public-cloud/01-aws/09-eks.md

```diff
@@ -0,0 +1,292 @@
+---
+title: "Create and Manage AWS EKS Cluster"
+metaTitle: "Creating new AWS EKS clusters on Palette"
+metaDescription: "Learn how to deploy and manage AWS EKS clusters with Palette"
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+
+
+
+# Create and Manage AWS EKS Cluster
+
+Palette supports creating and managing AWS Elastic Kubernetes Service (EKS) clusters deployed to an AWS account. This section guides you on how to create an AWS EKS cluster in AWS that is managed by Palette.

```

Comment:
`
Palette supports creating and managing AWS Elastic Kubernetes Service (EKS) clusters deployed to an AWS account. This section guides you on how to create an AWS EKS cluster in AWS that Palette manages.`